### PR TITLE
REVIEW: XMPP Newsletter February 2022

### PR DESCRIPTION
**Pull request by Echolon**
### Final release is planned for **5th of March 2021**.
---

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (xmpp:commteam@muc.xmpp.org?join)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/pull/1037/files)

:arrow_right: **Please do not just drop bare links** - rather try to make a short and decent formulated description (**<5 lines**) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 
---

This is our planed timeline for this draft:

XMPP Newsletter Timeline

Status:
- [x] Collection of news
- [x] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

File: content/blog/2022-03-05_newsletter.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+- German: [xmpp.org/de/blog](https://xmpp.org/de/blog/) and [anoxinon.de](https://anoxinon.de/blog/)
+- Italian: [nicfab.it](https://www.nicfab.it/)
+- Spanish: [xmpp.org/es/blog](https://xmpp.org/es/blog/)
+
+## XSF Announcements
+
+![XSF and Google Summer of Code 2022](/images/logos/GSoC_2022_Logo.png "XSF and Google Summer of Code 2022")
+
+- The XSF is planning to participate in [Google Summer of Code 2022 (GSoC)](https://xmpp.org/community/gsoc-2022/). If you are interested in participating as a student, mentor or as a project in general, please [add your ideas and reach out to us](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022).
+- All XSF members are called to vote in the currently on-going member applications.
```
Comment (SamWhited) [inline content/blog/2022-03-05_newsletter.md:26]: Maybe link to the wiki page?

File: content/blog/2022-03-05_newsletter.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+## Events
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Videos
```
Comment (SamWhited) [inline content/blog/2022-03-05_newsletter.md:40]: This section appears to be empty and can probably be removed.

File: content/blog/2022-03-05_newsletter.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+
+## Articles
+
+- JMP.chat [describes in their blog](http://blog.jmp.chat/b/2022-bidirectional-gateways) why gateways, and especially bidirectional gateways) with XMPP and other protocols, are so useful. [Their Newsletter](http://blog.jmp.chat/b/february-newsletter-2022) also announces the launch of worldwide calling and completion of their light rebrand.
```
Comment (SamWhited) [inline content/blog/2022-03-05_newsletter.md:45]: I'd scratch "light" and just call it a rebrand. It's just my personal preference to remove unnecessary adjectives.

File: content/blog/2022-03-05_newsletter.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+- Erik Bärwaldt, from the German "Linux Magazin" did [evaluate instant messenger including Conversations for XMPP](https://www.linux-magazin.de/ausgaben/2022/02/bitparade/) [DE].
```
Comment (SamWhited) [inline content/blog/2022-03-05_newsletter.md:47]: This sentence sounds odd. Remove the "did" and change "evaluate" to "evaluated" and the "messenger" to "messengers".

File: content/blog/2022-03-05_newsletter.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+## Software news
+
+### Clients and applications
+
+[Dino 0.3 has been released](https://dino.im/blog/2022/02/dino-0.3-release/): Video calls and conferences - encrypted and peer-to-peer. Dino now supports calls between two or more people!
+
+[Gajim development news](https://gajim.org/post/2022-02-24-development-news-february/): February brought a unified Groupchat Details window, which offers all the configuration knobs you need. Also this month: less memory leaks and improved OMEMO QR codes.
```
Comment (SamWhited) [inline content/blog/2022-03-05_newsletter.md:55]: s/Groupchat Details/group chat details/

File: content/blog/2022-03-05_newsletter.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
+Snikket published an important update for users of their [server software on ARM devices, including Raspberry Pi boards](https://snikket.org/blog/server-updates-for-arm/).
+
+[Tigase XMPP Server 8.2.0](https://github.com/tigase/tigase-server/releases/tag/tigase-server-8.2.0) has been released.
+
+## Libraries
+
+Mellium Co-op has released [v0.21.0](https://pkg.go.dev/mellium.im/xmpp@v0.21.0) and [v0.21.1](https://pkg.go.dev/mellium.im/xmpp@v0.21.1) of the [melium.im/xmpp](https://mellium.im/xmpp/) library for Go! Highlights for this release include basic support for 
+PubSub and PEP as well as In-Band Bytestreams. More information can be found in the [release announcement](https://opencollective.com/mellium/updates/new-release-mellium-im-xmpp-v0-21-0). Version 0.21.1 fixes a security issue that was discovered in the 
+websocket package and should be preferred when updating. For more information see [CVE-2022-24968](https://mellium.im/cve/cve-2022-24968/). [Mellium Dev Communiqué](https://opencollective.com/mellium/updates/dev-communique-for-february-2022):
```
Comment (SamWhited) [inline content/blog/2022-03-05_newsletter.md:83]: Something got changed here, the stuff at the bottom isn't about Communiqué, it's about the library. Can this be changed back to whatever was in the pad?

**Review by SamWhited — CHANGES_REQUESTED** [review]

File: content/blog/2022-03-05_newsletter.md (hunk 3 above)
Comment (Echolon) [reply content/blog/2022-03-05_newsletter.md:45]: Well, I would not treat this as unnecessary adjective, but I removed it.

File: content/blog/2022-03-05_newsletter.md (hunk 4 above)
Comment (Echolon) [reply content/blog/2022-03-05_newsletter.md:47]: non-native ¯\_(ツ)_/¯

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter February 2022 release

- Many thanks to everyone!!! This release to Adrien Bourmault (neox), alkino, anubis, Anoxinon e.V., Benoît Sibaud, cpm, Damian Sartori, emus, Ludovic Bocquet, Licaon_Kter, mathieui, MattJ, nicfab, seveso, Sam Whited, singpolyma, TheCoffeMaker, wurstsalat, xdelatour, Ysabeau for their help in creation, review and translation!

- :heart: :fireworks:  Next release is next year on 5th of April 2022!

Have a good time until then and see you soon!

**Pull request merged**